source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -6,6 +6,11 @@ protocol ACRContentHoldingViewProtocol {
     func applyPadding(_ padding: CGFloat)
 }
 
+protocol ErrorMessageHandlerDelegate: AnyObject {
+    func showErrorMessage(for view: NSView)
+    func hideErrorMessage(for view: NSView)
```

Comment:
This is supposed to be only for views which confirm to InputHandlingViewProtocol, right?

Reply:
Textfield doesn't require it to have the delegate, TextInputView conforms to it, and that's where the delegate is being called for text input

Reply:
See, as date, time, and number field don't just inherit from acrtextfield, they consume it. when we set the delegate of those fields, we actually are setting the delegate of textfield which doesn't conform to the protocol. so it would fail the check when you call hideErrorMessage(for: self) in ACRTextField

Reply:
I have made the change a few days ago in my branch. It works fine with the change.

Reply:
Where exactly are you calling hide errorMessage?

Reply:
In all views that conform to the protocol, not in ACRTextfield